src/index.css

```diff
@@ -4,3 +4,92 @@ body {
   display: flex;
   justify-content: center;
 }
+.restart {
+  width: 65px;
+  height: 20px;
+  color: white;
+  font-weight: bold;
+  font-size: 15px;
+  background-color: red;
+  border: none;
+}
+.game-title {
+  color: darkblue;
+  text-shadow: #FC0 1px 0 10px;
+  text-align: left;
+
+}
+.inputdev {
+  display: flex;
+   border: black;
+}
+.done {
+  display: flex;
+  font-weight: bold;
+  font-size: 15px;
+  margin-left: 10px;
+  color: white;
+  background-color: red;
+}
+.selected {
+  display: flex;
+  border: black solid;
+  width: 25px;
+  height: 20px;
+  color: black;
+  font-weight: bold;
+  font-size: 15px;
+}
+.board {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+}
+.yellowsquares {
+  background-color: yellow;
+  font-weight: bold;
+  font-size: 20px;
+  width: 20px;
+  height: 20px;
+  color: black;
+}
+.info-wrapper {
+  font-weight: bold;
+  font-size: 15px;
+  color: black;
+}
+.info-parms {
+  color: white;
+  border: solid;
+}
+.score {
+  display: flex;
+  justify-content: flex-end;
+  font-weight: bold;
+}
+.msgValid {
+  color: green;
+  font-weight: bold;
+  background-color: white;
+}
+.msgErr {
+  color: red;
+  font-weight: bold;
+  background-color: white;
+}
+.alphabet {
+  color: black;
+  font-weight: bold;
+  background-color: yellow;
```

Comment:
I would avoid using the css built in colors, they are usually very harsh and ugly, they are meant more for development debugging. You could probably find some nicer colors online